cli.js

```diff
@@ -48,7 +49,11 @@ var options = {
   delay: argv.delay,
   poll: argv.poll,
   killSignal: argv.killSignal,
-  outpipe: argv.outpipe
+  outpipe: argv.outpipe,
```

Comment:
Just a note, but when contributing to a repo try to stick with the existing code style 😄 E.g. no semi-colons here, no trailing commas. I'll quickly fix this up now.